### DISCUSSION TITLE
When will issue #636 on roadmap be implemented ?

### DISCUSSION THREAD

**Discussion by akcrisp** [community/community#137913] [Actions] [answered]
### roadmap item #636 allow secrets in called reusable workflow

Product Feedback

### Body

Guys can someone provide an eta as to when https://github.com/github/roadmap/issues/636 will be implemented ? I have literally just started using github actions and want to use reusable workflows but need to protect cert (image signing) from the calling actions / users being able to access the cert. I am amazed this feature hasn't been implemented already ! Yet all i see is it being pushed back with the last update sept 23. So a year ago.  This  feature needs adding asap.

**Comment by nebuk89** [comment, marked as answer]
Hey! Sadly we have no timeline for this one and as it kept moving around just put it into 'future' which means it's not this year for us at least. We will keep our roadmap updated as the source of truth for when for this :)

**Reply by akcrisp** [reply]
can you explain then how teams would go about creating a secure build pipeline if you can't prevent the caller repo from access the certificates ?? Perhaps I am missing something ?
